ESPixelStick/ESPixelStick.ino

```diff
@@ -133,11 +129,22 @@ void TestHeap(uint32_t Id)
 /** Arduino based setup code that is executed at startup. */
 void setup()
 {
+    if (!display.begin(SSD1306_SWITCHCAPVCC, SCREEN_ADDRESS))
+    {
+        for (;;)
+            ; // Don't proceed, loop forever
```

Comment:
We never loop forever. If it does not work, then print out a message and keep going. The oled is not a critical resource for operation. If you want to for a reboot then set the reboot request flag and move on.